Some tweaks to things on overmap map

comma opened the pull request:
Makes planetary missions use planet icons
Fixes a copypaste derp, blueriver mission never got name, marooned got it twice.
Made bearcat use shorter name for areas prefix.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

PsiOmegaDelta commented on maps/away/bearcat/bearcat.dm:
```diff
@@ -9,10 +9,11 @@
 	for(var/area/ship/scrap/A)
 		A.name = "\improper [name] - [A.name]"
 		GLOB.using_map.area_purity_test_exempt_areas += A.type
+	name = "[name], \a [name]"
```
"FTV Vagrant, a FTV Vagrant"?

comma committed:
Some tweaks to things on overmap map

Makes planetary missions use planet icons
Fixes a copypaste derp, blueriver mission never got name, marooned got it twice.
Made bearcat use shorter name for areas prefix.
